### PR TITLE
io: directory broswser

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**
Done with Directory Browser.

### CODE REVIEW

**Comment by mgenov** [comment]
Doesn't look like you are the only author of this ! 

screen shot 2016-05-30 at 2 11 35 pm

File: filebrowser/src/DirectoryBrowser.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+/**
+ * Created by clouway on 30.05.16.
+ */
+public class DirectoryBrowser {
+
+  public void listContent(String source) {
+    Path filePat= Paths.get(source);
+    if (isDirectory(filePat)) {
+      System.out.println(filePat.getFileName()+" = Directory.");
```
Comment (mgenov) [inline filebrowser/src/DirectoryBrowser.java]: Prints should be in the demo, not in the production code.

**Comment by AlexanderVladimirov9090** [comment]
Did not fix git config <- email and name.

**Comment by mgenov** [comment]
Ah... you changed the PC. OK

**Comment by AlexanderVladimirov9090** [comment]
Done.

File: dirbrowser/src/Demo.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by clouway on 30.05.16.
+ */
+public class Demo {
+  public static void main(String[] args) {
+    DirectoryBrowser browser = new DirectoryBrowser();
+
+    System.out.println(browser.listContent("/home/clouway/developmnet/path/WorkingPath"));
```
Comment (mgenov) [inline dirbrowser/src/Demo.java]: How can I print only directories which names are starting with "log_dir_"  from Demo ?

**Comment by AlexanderVladimirov9090** [comment]
Well I have to re-factor the code to do this.

**Comment by AlexanderVladimirov9090** [comment]
I think that way I can have full control of File array that is returned in method listContent() from DircetoryBrowser and Added new Class Checkpoint for checking if is file or dir.

**Pull request merged**